### DISCUSSION TITLE
Confused with Windows Pretixscan

### DISCUSSION THREAD

**Discussion by jxs714** [pretix/pretix#4406] [Q&A] [answered]
Hey guys,

A bit confused. I downloaded and installed the pretixscan windows app, but I do not see anything mentioning connecting to a camera to scan qr codes. Am I missing something? Does Pretixscan for windows allow using a webcam (or usb camera) to scan barcodes similar to the iOS and Android apps? The only option I see is to manually type the attendees name or ticket code.

Please advise, thanks.

**Comment by pc-coholic** [comment, marked as answer]
pretixSCAN Desktop is using (USB-)Barcode-Scanners in Keyboard Emulation mode - so you'll need to pickup such a device and configure and connect it to your computer.

Most USB-Barcodescanners will work, just make sure that they support Keyboard emulation and 2D Barcodes (QR being the important one). You can find our go-to devices at the bottom of this page: https://pretix.eu/about/en/hardware/scan

**Reply by jxs714** [reply]
Fantastic, thank you!
